### PR TITLE
JS_Questions Generation For The Game Mode

### PR DESCRIPTION
**Pull request by JustynaSmoliga**

### CODE REVIEW

File: src/app/Consts.js [hunk 1]
```diff
@@ -0,0 +1,5 @@
+export const GameModes = {
+    PEOPLE: "People",
+    STARSHIPS: "Starships",
+    VEHICLES: "Vehicles"
+}
```
Comment (kjreczek) [inline src/app/Consts.js:5]: The same const is on Filip's branch - FS_CreateFolders.

File: src/app/logic/questionsGenerator/QuestionsGenerator.js [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+
+
+async function generateQuestions(gameMode) {
+    let answers;
+    let rightAnswer;
+    let imageBlob;
+    if (gameMode === GameModes.PEOPLE) {
+        const allowedIds = [];
+        for (let i = 1; i < 83; i++) {
+            if (i === 17) continue;
```
Comment (kjreczek) [inline src/app/logic/questionsGenerator/QuestionsGenerator.js:15]: I would define what are numbers 83 and 17 as a consts

File: src/app/Consts.js (hunk 1 above)
Comment (fszwedo) [reply src/app/Consts.js:5]: I'll adjust my functions to use Justyna's consts during the next 'sprint' :)

File: src/app/logic/questionsGenerator/QuestionsGenerator.js (hunk 2 above)
Comment (JustynaSmoliga) [reply src/app/logic/questionsGenerator/QuestionsGenerator.js]: Done

**Review by juliakozlowska — APPROVED** [review]

**Pull request merged**